sql/hive/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -1,140 +0,0 @@
-/*
```

Comment:
Just so my head's straight here, these are strictly duplicated so can just be removed?